index.html

```diff
@@ -0,0 +1,217 @@
+                        <img src="./images/lang.png" alt="languages mastered">
+                    </div>
+                    <h4>Language</h4>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>Html </li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="framework tech-sk">
+                    <div class="frm">
+                        <img src="./images/frame.png" alt="frameworks used">
+                    </div>
+                    <h4>Framework</h4>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Webpack</li>
+                    </ul>
+                </div>
+                <div class="skill-set tech-sk">
+                    <div class="sk">
+                        <img src="./images/skillz.png" alt="skills">
+                    </div>
+                    <h4>Skills</h4>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="contact-section" id="contact">
+        <h2 id="greeting">Hello</h2>
+        <p>I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.</p>
+        <form action="/send-https://formspree.io/f/mpzbbknz" method="POST" class="contact-form" id="contact-details">
+            <label>
+                <input type="text" id="name" name="name" placeholder="Full name" required maxlength="30">
+            </label>
+            <label>
+                <input type="email" id="email" name="email" placeholder="Email address" required maxlength="30">
+            </label>
+            <select name="continent" id="continent">
+                <option>Africa</option>
+                <option>Asia</option>
+                <option>Antactica</option>
+                <option>Australia</option>
+                <option>Europe</option>
+                <option>North America</option>
+                <option>South America</option>
+            </select>
+            <label>
+                <textarea id="message" maxlength="500" name="message" required placeholder="Enter text here" cols="30"
+                    rows="10" required></textarea>
+            </label>
+            <div class="btn-container" id="btn">
+                <button type="submit" class="submit button">Get in touch</button>
+                <small class="error-message hide">Please fill in lower case</small>
+            </div>
+        </form>
```

Comment:
- [ ] Although it is very brilliant 🥇  of you to add a continent ``select`` tag, I think it would be better to stick to the template as it is with the same tags, font, colors, and spacing because it is very important as a front end developer to be able to create a designed UX as it is. ☺️ 

### Figma design 
![design](https://user-images.githubusercontent.com/102636489/225763621-f42d308d-32aa-44b3-a992-b6dd75e47c96.png)

### Your design
<img src="https://user-images.githubusercontent.com/102636489/225763696-faaab51f-e9db-41e9-b34b-1f21ac669d40.png" style="width: 50%;">